resources/views/page_select.blade.php

```diff
@@ -0,0 +1,8 @@
+@extends('support::layout')
```

Comment:
Really don't think you need this.  Why would we ever ask someone to select a page?  Bad url => 404 error.